mailviews/tests/urls.py

```diff
@@ -1,13 +1,21 @@
-try:
-    from django.conf.urls import include, patterns, url
-except ImportError:
-    from django.conf.urls.defaults import include, patterns, url
+from mailviews.utils import is_django_version_greater
+
+if is_django_version_greater('1.9'):
+    from django.conf.urls import include, url
+else:
+    from django.conf.urls import patterns, include, url
+
 
 from mailviews.previews import autodiscover, site
 
 
 autodiscover()
 
-urlpatterns = patterns('',
-    url(regex=r'', view=site.urls),
-)
+if is_django_version_greater('1.9'):
+	urlpatterns = [
+		url(regex=r'', view=site.urls)
+	]
+else:
+	urlpatterns = patterns('',
```

Comment:
You should be able to drop the conditional and just use the list syntax for these urls.